tox.ini

```diff
@@ -1,5 +1,16 @@
 [tox]
-envlist = py26, py27, py32, py33, py34
+envlist = py26, py27, py32, py33, py34, flake, flake_docs
 
 [testenv]
 commands = python tests.py
+
+[testenv:flake]
+deps = flake8
+exclude = docs/conf.py
+doctests = no
+
+[testenv:flake_docs]
+deps = flake8
+exclude = docs/conf.py
+doctests = yes
+ignore = F821
```

Comment:
Ya I think we both realized this at the exact same time. I went ahead and just did the configuration in the Makefile for now. It's not the best solution but it's cleaner than what we had.